browser_use/browser/context.py

```diff
@@ -437,9 +437,23 @@ async def _create_context(self, browser: PlaywrightBrowser):
 		# Load cookies if they exist
 		if self.config.cookies_file and os.path.exists(self.config.cookies_file):
 			with open(self.config.cookies_file, 'r') as f:
-				cookies = json.load(f)
-				logger.info(f'🍪  Loaded {len(cookies)} cookies from {self.config.cookies_file}')
-				await context.add_cookies(cookies)
+				try:
+					cookies = json.load(f)
+
+					valid_same_site_values = ['Strict', 'Lax', 'None']
+					for cookie in cookies:
+						if 'sameSite' in cookie:
+							if cookie['sameSite'] not in valid_same_site_values:
+								logger.warning(
+									f"Fixed invalid sameSite value '{cookie['sameSite']}' to 'None' for cookie {cookie.get('name')}"
+								)
+								cookie['sameSite'] = 'None'
+
+          logger.info(f'🍪  Loaded {len(cookies)} cookies from {self.config.cookies_file}')
```

Comment:
I think spaces on this line should be tabs

(make sure to run `ruff check --fix; ruff format` before commit)